### DISCUSSION TITLE
IE11 completely deprecated in v6?

### DISCUSSION THREAD

**Discussion by ghost** [remarkjs/remark#682] [Q&A] [answered]
There's a note at the end of the v6 release notes that IE11 is deprecated. It's not clear from the note whether an external polyfill will make the v6 version of this library work in IE11, or if IE11 is just completely unsupported going forward.

I'd love to bail on IE11, but alas there are plenty of corporate environments that still rely, and will continue to rely, on that browser.

**Comment by ChristianMurphy** [comment, marked as answer]
[version 6 of react-markdown](https://github.com/remarkjs/react-markdown/blob/main/changelog.md#600---2021-04-15)?
Both transpiling and polyfills would be needed. Babel preset env can take care of both https://babeljs.io/docs/en/babel-preset-env

**Reply by ghost** [reply]
So if I have this in my `package.json` file, based on that documentation, then I should be good?

```
  "browserslist": [
    ">0.2%",
    "not dead",
    "not ie < 11",
    "not op_mini all"
  ]
```

**Reply by ChristianMurphy** [reply]
checking with

```sh
$ npx browserslist ">0.2%, not dead, not ie < 11, not op_mini all"
and_chr 89
and_ff 86
and_uc 12.12
chrome 89
chrome 88
chrome 87
chrome 86
chrome 85
edge 89
edge 88
firefox 87
firefox 86
ie 11
ios_saf 14.0-14.5
ios_saf 13.4-13.7
ios_saf 13.3
ios_saf 12.2-12.4
opera 73
safari 14
safari 13.1
safari 8
samsung 13.0
```

assuming the above are the browsers you want to support, yes

**Reply by ghost** [reply]
Oh cool! I didn't know that feature existed
